### DISCUSSION TITLE
Can someone give me idea or tutorial how can i deploy my webpack bundle to local servers

### DISCUSSION THREAD

**Discussion by Nikunj-bisht** [callstack/repack#513] [Q&A]
## Ask your Question

How i can i deploy my mini app to the server if someone could help me with this that what  i have to deploy step wise. thankyou

**Comment by heradonbenin** [comment]
hello did you found any solution?

**Comment by jbroma** [comment]
Please see [Super-App Showcase](https://github.com/callstack/super-app-showcase), we're hosting mini-apps on GitHub there, it might give you some insights on how do this. We don't have any tutorials on how do this right now, but that might change in the future.

**Reply by trdquang** [reply]
Hi @jbroma, we are going to build a node server to serve mini app bundle as I see github packages are static files, do you think it is a valid approach. 
Besides that, we are worrying about security issues. I know that we are using code signing to protect mini apps, are there any things we can do to enhance the security? 

As we intent to propose repack as solution for our customers. 

Thanks in advance.

**Comment by mlakmal** [comment]
you can use aws cloudfront and s3 to store miniapp bundles. this will allow you to easily manage versioning per bundle release and falling back to previous versions without maintaining custom mechanism.

ex: miniapps.xyz.com/miniappX/v1.0.0/{bundle_files}

setup miniapps.xyz.com domain for cloudfront and setup cloudfront to map requests to internal s3 bucket. then you just need to upload the bundles for each miniapp to specific folder path in s3 bucket.

you can maintain a rest api to specify which versions of the miniapp is needed by host application. and call this rest api through re.pack resolver to identify and create remote url paths...
